iob_soc.py

```diff
@@ -302,19 +277,24 @@ def __init__(self, *args, **kwargs):
                 "USE_MUL_DIV": USE_MUL_DIV,
                 "USE_EXTMEM": USE_EXTMEM
             },
-            io_connections={
-                "clk_i": clk,
-                "arst_i": cpu_reset,
-                "cke_i": cke,
-                "boot_i": boot,
-                "trap_o": trap,
+            connect={
+                "clk": get_wire_from_group("clk_en_rst", "clk"),
```

Comment:
connect the whole group, not individual ports 

ports cannot have any logic or concatenations or part selections

Reply:
Removed individual port connections in commit [e789415](https://github.com/IObundle/iob-soc/pull/813/commits/e7894154e966b1e9bd2984d058d37fc785a9da72).
In this commit, I created a dedicated `cpu_clk_en_rst` wire group for the CPU.
In the future, we may use the standard `clk_en_rst` group, but for now, without refactoring the iob_soc.v and iob_picorv32.v modules, we use the dedicated `cpu_clk_en_rst` wire group.